SparkPost/2web2ui
# FE-1159 - Add UI flag to prevent rendering domains link

[FE-1159](https://sparkpost.atlassian.net/browse/FE-1159)

### What Changed
- Adds missing condition on the domains link in the Hibana navigation

### How To Test
1. Sign in to the app with an account that does not have the flag `allow_domains_v2`
2. Enable Hibana
3. Click on "Configuration"
4. Verify you are routed to the old Sending Domains page
5. Sign in to the app with an account that _does_ have the flag `allow_domains_v2`
6. Click on "Configuration"
7. Verify that you are routed to the new domains page


### To Do
